core/src/main/java/org/keycloak/jose/jwk/ECPublicJWK.java

```diff
@@ -65,4 +66,26 @@ public String getY() {
     public void setY(String y) {
         this.y = y;
     }
+
+    @JsonIgnore
+    @Override
+    public <T> T getOtherClaim(String claimName, Class<T> claimType) {
```

Comment:
Just out of curiosity, would Keycloak project introduces the latest switch statement format as follows ?

```
        Object claim = switch (claimName) {
            case CRV -> getCrv();
            case X -> getX();
            case Y -> getY();
            default -> null;
        };
```

Reply:
Thanks for the suggestion, however unfortunately, I think that these might be problematic as this format is supported since Java 12, but those classes from the `core` module are still synced to keycloak/keycloak-client repository, which is supposed to work with Java 11. So I've rather did not applied this recommendation to avoid possible issues with keycloak-client compatibility.